pincer/objects/app/command.py

```diff
@@ -81,9 +88,13 @@ class AppCommandOption(APIObject):
     name: str
     description: str
 
-    required: APINullable[bool] = False
+    required: APINullable[bool] = MISSING
```

Comment:
From [the documentation](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-structure):
`required? | boolean | if the parameter is required or optional--`**`default false`**
:thinking:

Reply:
I guess I should set it to False so its an easier change if they make it required in API v10?